### DISCUSSION TITLE
NPM RUN BUILD > Build error occurred [ null, type: 'Array', stack: null ] Without Error Logs

### DISCUSSION THREAD

**Discussion by msanmaz** [vercel/next.js#34289] [Help]
Hi I've been working on a website, before it was deploying and I had no problem running npm run build, However all of a sudden it started to throw an error, I made a search about it, I updated my version of next and react, I tried to update all packages to the latest version but it still wouldnt work. Any idea how to make deployment work.

- I tried to clear npm cache,
- Removed node modules and reinstall
- update all the dependency
- update versions of next and node
still having the same error

<img width="548" alt="Screenshot 2022-02-13 at 13 47 07" src="https://user-images.githubusercontent.com/47824518/153756091-c3cb4b53-5d60-4ff6-8e70-42de16cb1045.png">

<img width="716" alt="Screenshot 2022-02-13 at 12 03 44" src="https://user-images.githubusercontent.com/47824518/153752212-0a3a820f-a7f1-45b7-9912-ccb142eb6e6b.png">

**Comment by msanmaz** [comment]
my package.json as below

{
  "name": "attila-homes",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "build-css": "cross-env NODE_ENV=production tailwindcss build app.css -o styles.css",
    "dev-css": "cross-env NODE_ENV=development tailwindcss build app.css -o styles.css",
    "next-dev": "next dev",
    "dev": "concurrently \"npm run next-dev\" \"npm run dev-css\"",
    "next-build": "next build",
    "build": "npm run next-build",
    "start": "NODE_ENV=development node server.js"
  },
  "dependencies": {
    "@reduxjs/toolkit": "^1.6.2",
    "aos": "^2.3.4",
    "autoprefixer": "^10.0.4",
    "concurrently": "^5.3.0",
    "cross-env": "^7.0.2",
    "emailjs-com": "^3.2.0",
    "firebase": "^8.1.2",
    "hamburger-react": "^2.4.0",
    "jquery": "^3.5.1",
    "next": "^10.0.3",
    "next-pwa": "^5.0.4",
    "next-redux-wrapper": "^7.0.5",
    "next-seo": "^4.28.1",
    "postcss-flexbugs-fixes": "^5.0.2",
    "postcss-preset-env": "^7.3.3",
    "react": "17.0.1",
    "react-dom": "17.0.1",
    "react-google-maps": "^9.4.5",
    "react-image-gallery": "^1.0.8",
    "react-redux": "^7.2.6",
    "react-scroll": "^1.8.1",
    "react-select": "^3.1.1",
    "react-spinners": "^0.9.0",
    "sass": "^1.32.2",
    "scss": "^0.2.4",
    "tailwindcss": "^2.0.1"
  }
}

**Comment by msanmaz** [comment]
Now all the other errors disappeared, during the npm run build it starts to build when it comes to collecting page data.% it just said; Build error occured and nothing else, I really dont know what to do as it is doesnt point out any indication on the error. I didnt make any changes after my last deployed commit, however it stopped working

**Comment by msanmaz** [comment]
I was trying to troubleshoot the problem by checking each individual page, and I've found out after I deleted all pages that uses getstaticprops and getstaticpaths, my deployment was successful, however I've never changed the use of getstaticprops and patch but all of a sudden it start fails my deployment, any help would be appreciated

**Comment by hritu8** [comment]
i want to implement null checks warnings in build . how to implement this
